src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -348,7 +347,7 @@ bool WriteLog<I>::initialize_pool(Context *on_finish, pwl::DeferredContexts &lat
       on_finish->complete(-EINVAL);
       return false;
     }
-    this->m_log_pool_actual_size = D_RO(pool_root)->pool_size;
+    this->m_log_pool_config_size = D_RO(pool_root)->pool_size;
```

Comment:
This can also fix the bug in line:366.
`size_t effective_pool_size = (size_t)(this->m_log_pool_config_size * USABLE_SIZE);`